hpc-coveralls.cabal

```diff
@@ -60,6 +60,7 @@ library
     cmdargs >= 0.10,
     curl >= 1.3.8,
     hpc >= 0.6,
+    process >= 1.1.0.1,
```

Comment:
It's just the first version I could confirm to have `readProcess`, and so seemed to be the logical lower bound.